jumanji/environments/logic/rubiks_cube/viewer.py

```diff
@@ -103,9 +110,10 @@ def _get_fig_ax(self) -> Tuple[plt.Figure, List[plt.Axes]]:
                 fig.show()
         return fig, ax
 
-    def _draw(self, ax: List[plt.Axes], state: State) -> None:
-        i = 0
-        for face in Face:
+    def _draw(self, ax: List[plt.Axes], state: State) -> List[AxesImage]:
+        images = list()
```

Comment:
nit, bit more pythonic
```suggestion
        images = []
```

Reply:
Could have sworn that `list()` was preferred for some pep reason, but looking into think you are correct. Must have imagined it 😂!